### DISCUSSION TITLE
password column in 'show fds' in 1.23.1 gone

### DISCUSSION THREAD

**Discussion by achix** [pgbouncer/pgbouncer#1150] [General]
Hello, I noticed that till 1.22.1 , 'show fds' was showing the actual password of the user connected, in 1.23.1 this is now empty. Is this change intentional ?

**Comment by JelteF** [comment]
I don't think it was intentional. It's probably related to the changes made in #1052. Could you create an issue for this? Also, out of curiosity what were you using this for that you noticed?

**Reply by achix** [reply]
We never used this, this is just an observation. :)
